lnrpc/lightning.proto

```diff
@@ -4231,6 +4231,10 @@ message DeleteAllPaymentsRequest {
     Only delete failed HTLCs from payments, not the payment itself.
     */
     bool failed_htlcs_only = 2;
+
```

Comment:
maybe the `DeleteAllPayments` should be renamed to `DeletePayments` for clarity - future PRs tho.

Reply:
Unfortunately we can't really do that without breaking older RPC clients... But yeah, would've been more precise.

Reply:
I thought of the backward compatibility as well 👍